apps/src/templates/sectionProgressV2/IconKey.jsx

```diff
@@ -0,0 +1,64 @@
+import React, {useState} from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import Button from '@cdo/apps/templates/Button';
+import color from '@cdo/apps/util/color';
+import LevelTypesBox from './LevelTypesBox';
+import TeacherActionsBox from './TeacherActionsBox';
+import AssignmentCompletionStatesBox from './AssignmentCompletionStatesBox';
+
+export const NOT_STARTED = 'Not started';
```

Comment:
Since these are passed around as static strings and checked against, I think it's safer and faster to use an enum type instead of string constants.